CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
```

Comment:
In your commit message:

`s/To be use with/To be used with/`

`s/when migration become too long/when the migration becomes too long/`

`s/We can provide a sql dump of the schema to then load it/We can provide a sql dump of the schema and then load it`